xml/admin_gui_dashboard.xml

```diff
@@ -2327,6 +2413,130 @@ Oct 04 07:48:53 doc-ses6min4 systemd[1]: Started Ceph rbd mirror daemon.
 </screen>
     </tip>
    </sect3>
+   <sect3 xml:id="cert-sign-custom-CA">
+     <title>Certificates Signed with a Custom CA</title>
+     <para>
+       The following procedure needs to be followed once to create the root CA.
+     </para>
+     <note>
+        <para>
+          This is the key used to sign the certificate requests. Anyone
+           holding this can sign certificates on your behalf.
+         </para>
+      </note>
+     <procedure>
+       <step>
+         <para>
+           Create the Root Key:
+         </para>
+<screen>
+&prompt.cephuser;openssl genrsa -des3 -out rootCA.key 4096
+</screen>
+        <note>
+          <para>
+            If you want a non-password protected key, remove the
+            <option>-des3</option> option.
+          </para>
+        </note>
+       </step>
+       <step>
+         <para>
+           Create and self-sign the root certificate:
+         </para>
+<screen>
+&prompt.cephuser;openssl req -x509 -new -nodes -key rootCA.key -sha256 -days 1024 -out rootCA.crt
+</screen>
+       </step>
+     </procedure>
+     <para>
+       The following procedure needs to be followed for each server
+       that needs a trusted certificate from our CA.
+     </para>
+     <procedure>
+       <step>
+         <para>
+          Create the certificate key:
+        </para>
+<screen>
+&prompt.cephuser;openssl genrsa -out mydomain.com.key 2048
+</screen>
+	<para>
+           The certificate signing request is where you specify the details for
+           the certificate you want to generate. This request is processed by
+           the owner of the Root Key to generate the certificate.
+         </para>
+       </step>
+       <step>
+         <important>
+           <para>
+             When creating the certificate signing request, it is important to specify
+             the <literal>Common Name</literal> providing the IP address or
+             domain name for the service, otherwise the certificate cannot
+             be verified.
```

Comment:
This builds with the ` 2.` for the second step being followed directly by/sitting right next to the `Important` admonition. It looks slightly confusing and I had to looks twice before I saw that the second step is to create a CSR. I would suggest to add "Create a certificate signing request." just before the admonition, to clarify this.